source/community/reactnative/src/components/actions/select-Action.js

```diff
@@ -51,7 +45,39 @@ export class SelectAction extends React.Component {
 				this.toggleVisibilityForElementWithID(this.props.selectActionData.targetElements);
 				break;
 			default:
+				this.onCustomActionCalled();
```

Comment:
As per the AC schema, ShowCard action type is not supported by selectAction.
Have to add that condition under default section instead of allowing all.

Reply:
Added the condition